scipy/stats/tests/test_continuous.py

```diff
@@ -478,7 +489,7 @@ def check_cdf2(dist, log, x, y, result_shape, methods):
                 or dist._overrides('_logccdf_formula')):
             methods.add('log/exp')
 
-    ref = dist.cdf(y) - dist.cdf(x)
+    ref = np.where(x > y, 0.0, dist.cdf(y) - dist.cdf(x) + dist.pmf(x))[()]
```

Comment:
I had to change `ref` here and elsewhere to account for

1. The change in behavior when `x > y`.
2. The discrete case. Fortunately `pmf(x)` is defined and equals zero everywhere for continuous distributions.